ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1574,8 +1604,24 @@
 
             # Outbound route-map name.
             route_map_out: <str>
+
+            # Inbound RCF function name with parenthesis.
+            # Example: MyFunction(myarg).
+            rcf_in: <str>
+
+            # Outbound RCF function name with parenthesis.
+            # Example: MyFunction(myarg).
+            rcf_out: <str>
         neighbor_default_encapsulation_mpls_next_hop_self:
           source_interface: <str>
+        redistribute_routes:
```

Comment:
I dont think we need this as applying redistribute route under address family will redirect to global config thats the reason its not added for route map also.
cc: @ClausHolbechArista

Reply:
I think it depends on the specific routing protocols and capabilities, but for now I think we can just remove this and wait for a specific request.

Reply:
Removed redistribute_routes from vpn-ipv4/ipv6